test/e2e/parallel/nstemplatetier_test.go

```diff
@@ -469,6 +469,41 @@ func TestTierTemplateRevision(t *testing.T) {
 
 	})
 
+	t.Run("when updating one tiertemplate the revisions field should be cleaned up from old entries", func(t *testing.T) {
```

Comment:
Similar to Host Operator, should we also have a check where we are creating ttr.?

Reply:
If I'm well understanding the scenario, it's the main one, covered also at the beginning of the test : https://github.com/codeready-toolchain/toolchain-e2e/blob/92500ba037043c731472701e1a3b6ee907dfddcc/test/e2e/parallel/nstemplatetier_test.go#L369-L408